201-key-vault-with-logging-create/azuredeploy.json

```diff
@@ -160,9 +160,10 @@
       },
       "resources": [
         {
-          "type": "Microsoft.KeyVault/vaults/providers/diagnosticsettings",
-          "name": "[concat(parameters('keyVaultName'), '/Microsoft.Insights/service')]",
+          "scope": "[format('Microsoft.KeyVault/vaults/{0}', parameters('keyVaultName'))]",
```

Comment:
another nested child where scope is not allowed

Reply:
@bmoore-msft - should I just refactor this so that it isn't a nested child? It's not currently taking advantage of the shortened nested child type syntax.

Reply:
yes please